cmd/launcher/launcher.go

```diff
@@ -216,6 +217,13 @@ func runLauncher(ctx context.Context, cancel func(), slogger, systemSlogger *mul
 		}
 	}
 
+	s, err := startup.NewStartupDatabase(ctx, k)
+	if err != nil {
+		k.Slogger().Log(ctx, slog.LevelWarn, "could not create startup db", "err", err)
```

Comment:
We might want to bail out if we can't make this. Not sure

Reply:
Since it's not critical yet, I figured don't bail out now, and update in the future when it becomes critical -- but it probably makes more sense to just bail out now so I don't have to remember to update it later